### DISCUSSION TITLE
Power supply survey

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#61] [Polls]
What is your prefered power supply solution ?
_The AC version was discarded because some users were unaware and did not follow basic security principles._

**Comment by nstadigs** [comment]
With USB-C people can just visit their local Ikea and get a SJÖSS charger. It's rated for 45W/3A

**Comment by mkaiser** [comment]
What about 24V?

**Reply by nliaudat** [reply]
24V is much like 230V. 
It's better to handle it with relays
Do you have a DC proportionnal valve sample which works under 24V ?

**Reply by mkaiser** [reply]
> 24V is much like 230V.
I disagree here :)

24V is standard for in-house cabling in smart homes (Homematic IP, KNX (altough current controlled...) and Loxone are using it. 

In my "heating circuit distributor" (never used this english word, I hope it is right...), I have a KNX-like YSTY cable with GND/ +24V  and some cables for data communication. 

This is why it would be great for me to directly use the available 24V. In several other projects I used the switching regulators K7803-500R3 with an input range from 5..26V to 3.3V (500mA) to power microcontrollers directly. 

How much power will the valves typically drain? 

I could not find the schematics - will you put them back online after your reworks?

> Do you have a DC proportionnal valve sample which works under 24V ?

No, I would like to use the Homematic valves.

**Reply by nliaudat** [reply]
I tried  XL1509 module ( 3.3V to 37V ) and it's working fine to power the board in 5V with upper voltage. 
I was proposing it in th v59 release, but nobody needed such voltage. 
If many people ask for >5v power voltage, I make it. 
In the meanwhile  XL1509 module is a low cost solution for your needs. 
Regards

**Reply by nliaudat** [reply]
An Homematic valve drain 120mA in their datasheet. Really, it's over 500mA in near endstop position

**Comment by DerMo92** [comment]
I'm still using 230V and plan to switch to a 5V DIN-Rail supply. But the version I have, has not the best connection points. 
(I have all PCB in one place in on panel cabinet and extended the RJ11 to the Valves in each floor.)

**Reply by nliaudat** [reply]
Sorry, I do not understant your needs. 
Are you valves 230V or your mainboard ?
What is you expectations ?

**Reply by DerMo92** [reply]
I use 230V with a Version of your PCB the Valves are VMoDs with RJ11. I plan to solder 5V to it using a DIN Rail Power supply. 
I hope this was better understandable.
